<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.gt.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" tests="5" skipped="0" failures="0" errors="0" timestamp="2022-12-29T00:07:39" hostname="nb-005045" time="0.027">
  <properties/>
  <testcase name="givenAValidCommand_whenGatewayThrowsRandomException_shouldReturnAException()" classname="com.gt.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.005"/>
  <testcase name="givenAValidInactivateCommand_whenCallsUpdateCategory_shouldReturnInactiveCategoryId()" classname="com.gt.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.007"/>
  <testcase name="givenACommandWithInvalidID_whenCallsUpdateCategory_shouldReturnNotFoundException()" classname="com.gt.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.004"/>
  <testcase name="givenAValidCommand_whenCallsUpdateCategory_shouldReturnCategoryId()" classname="com.gt.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.003"/>
  <testcase name="givenAInvalidName_whenCallsUpdateCategory_thenShouldReturnDomainException()" classname="com.gt.admin.catalogo.application.category.update.UpdateCategoryUseCaseTest" time="0.003"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
